### ISSUE TITLE
No headings exist 

### ISSUE THREAD

**Issue by anna-guo-level-access** [anna-guo-level-access/a#33]
**Rule description**  
The body element does not contain elements that form a heading structure (i.e. h1-h6 elements; or elements given a role=heading).

**Tool name**  
Access Engine

**Severity**  
High

**WCAG Success criteria**  
[1.3.1 Info and Relationships - Level A](https://www.w3.org/WAI/WCAG22/Understanding/info-and-relationships)

**Why it matters**  
Without a proper heading structure, informational hierarchy may not be clearly communicated to users who cannot perceive visual formatting of headings. Users of screen readers who are blind or low vision who navigate using headers will not be able to navigate as easily without these navigational aids.

**What to do**  
Use heading elements for any elements that use visual effect to convey a content/section heading. HTML heading elements such as <code>h1</code>,<code>h2</code>,<code>h3</code>,<code>h4</code>,<code>h5</code>, and <code>h6</code> need to be used to specifically mark up page content that visually appears as a heading.  

The ARIA role of <code>heading</code> may also be used with the <code>aria-level</code> attribute to specify heading structure when semantic elements are not available.

**Task in the Level Access Platform**  
https://tenantb.localhost-dashboard.essentialaccessibility.com/my-tasks/6a3c613b43450165564cc432/6a4bac6f2b365496add97ba3

**Comment by anna-guo-level-access** [comment]
Task pushed to GitHub Issues by Anna Guo from Level Access Platform.

Posted by Anna Guo via Level Access

**Comment by anna-guo-level-access** [comment]
Task pushed to GitHub Issues by Anna Guo from Level Access Platform.

Posted by Anna Guo via Level Access

**Comment by anna-guo-level-access** [comment]
<a href="https://tenantb.localhost-dashboard.essentialaccessibility.com/tasks/6a4bac6f2b365496add97ba3/attachments/6a4bac6f2b365496add97ba1/download" target="_blank" rel="noopener noreferrer">Linked Findings.csv</a>
